scripts/combine_control_files.py

```diff
@@ -0,0 +1,35 @@
+# Script to generate ext_index.json metadata file
+# that stores content of the control files and location of extension archives
+#  for all extensions in /extensions subdir.
+import json
+import os
+import sys
+from os.path import isdir, join
+
+pg_version = sys.argv[1]
+assert pg_version in ("v14", "v15")
+
+build_numbers = {}
+with open("build_numbers.txt", "r") as f:
+    for line in f.readlines():
+        ext_name, build_num = line.split(" ")
+        build_numbers[ext_name] = build_num.strip()
+
+ext_index = {}
+os.chdir("extensions")
```

Comment:
The path do this directory should be parametrised

Reply:
see slack conversation